universal-application-tool-0.0.1/app/controllers/admin/AdminProgramBlockPredicatesController.java

```diff
@@ -183,7 +183,9 @@ private PredicateValue parsePredicateValue(
         // of the method with `values`.
         switch (operator) {
           case ANY_OF:
+          case IN:
```

Comment:
suggestion (blocking): I think we should either add *all* the right-hand-side list operators here, or else only keep `IN` and `NOT_IN`. The reason I had only kept `ANY_OF` and `NONE_OF` here in the first place instead of all of them was because I was only trying to include right-hand-side list operators that were relevant to left-hand-side string operators, if that makes sense. Unfortunately I clearly chose the wrong two operators. But anyway, it might be safer in some ways to include all of the list operators. Even though in theory, by this point in the code, `IN` and `NOT_IN` should be the only ones of concern.

PS THANK YOU FOR FIXING THIS

Reply:
Ah that makes sense, thank you for clarifying! I added all the operators and updated the comment to explain that only IN and NOT_IN are expected here